app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
Cool. If that's the case I think we should be good. Will change it to a file manager implementation.